### PR TITLE
Fix audit with `pnpm audit --prod --fix`

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: .github/workflows/audit.yml [hunk 1]
```diff
@@ -22,4 +22,4 @@ jobs:
       - uses: pnpm/action-setup@v4
         with:
           version: 9.4
-      - run: pnpm audit
+      - run: pnpm audit --prod
```
Comment (nop33) [inline .github/workflows/audit.yml:25]: @h0ngcha0 don't you think it makes sense to only audit packages that are used in production

https://pnpm.io/cli/audit#--prod--p

Comment (h0ngcha0) [reply .github/workflows/audit.yml:25]: Sorry for being late, would it be an issue if we audit all packages? `devDependencies` might affect our dev environment so if we can maybe we should audit as well, WDYT?

Comment (nop33) [reply .github/workflows/audit.yml:25]: ok, fair enough!

**Comment by polarker** [comment]
@nop33 It seems only webpack needs to be patched for dev deps. Can you give it a try?

**Pull request merged**